### PR TITLE
AVRO-3450: Document editor support for IDL

### PR DESCRIPTION
**Pull request by opwvhk**
Make sure you have checked _all_ steps below.

### Jira

- [X] My PR addresses the following [Avro Jira](https://issues.apache.org/jira/browse/AVRO/) issues and references them in the PR title. For example, "AVRO-1234: My Avro PR"
  - https://issues.apache.org/jira/browse/AVRO-3450
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Tests

- [X] My PR ~adds the following unit tests __OR__~ does not need testing for this extremely good reason: it add documentation

### Commits

- [X] My commits all reference Jira issues in their subject lines. In addition, my commits follow the guidelines from "[How to write a good git commit message](https://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [X] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain Javadoc that explain what it does

### CODE REVIEW

File: doc/content/en/docs/next/IDL Language/_index.md [hunk 1]
```diff
@@ -374,3 +374,60 @@ protocol Simple {
 }
 ```
 Additional examples may be found in the Avro source tree under the `src/test/idl/input` directory.
+
+## IDE support
+
+There are several editores and IDEs that support Avro IDL files, usually via plugins.
```
Comment (martin-g) [inline doc/content/en/docs/next/IDL Language/_index.md:380]:
```suggestion
There are several editors and IDEs that support Avro IDL files, usually via plugins.
```

File: doc/content/en/docs/next/IDL Language/_index.md [hunk 2]
```diff
@@ -374,3 +374,60 @@ protocol Simple {
...
+and [GitHub](https://github.com/Jason3S/vscode-avro-ext)
+
+### Atom.io
+
+atom-language-avro 0.0.13 was released on 14 August 2015. It provides syntax highlighting.
+
+It is available as [Atom.io package](https://atom.io/packages/atom-language-avro)
+and [GitHub](https://github.com/jonesetc/atom-language-avro)
+
+### ViM
```
Comment (martin-g) [inline doc/content/en/docs/next/IDL Language/_index.md:427]:
```suggestion
### Vim
```

**Review by martin-g — APPROVED** [review]

File: doc/content/en/docs/next/IDL Language/_index.md (hunk 1 above)
Comment (opwvhk) [reply doc/content/en/docs/next/IDL Language/_index.md]: Hrm... you're right.  In my memory there was a second capital, but the website of the editor says I'm wrong...

**Review by martin-g — APPROVED** [review]

**Comment by martin-g** [comment]
Thank you, @opwvhk !

**Pull request merged**